webpack.config.js

```diff
@@ -9,9 +9,10 @@ const mode = process.env.NODE_ENV;
 const dev = mode === "development";
 
 const alias = {
+  "~": path.resolve(__dirname, "src/"),
```

Comment:
These shortcuts are great!

Reply:
Thanks! Yeah one benefit of using webpack's alias so you don't have to go crazy with `../`'s